.github/workflows/sdk_protos_map.csv

```diff
@@ -420,6 +420,7 @@ data_sync,StreamingDataCaptureUpload,streaming_data_capture_upload,,
 
 ## MLTraining
 mltraining,SubmitTrainingJob,submit_training_job,,
+mltraining,SubmitCustomTrainingJob,submit_custom_training_job,,
```

Comment:
@andf-viam How can I generate the markdown from on this pr? want to put SubmitCustomTrainingJob from docs/build/program/apis/ml-training-client.md in right place

Reply:
Right. This resource has been converted to autogen, so indeed it's as you say: 

- Update the mapping file with the proto and at least one method. You have it correct above! (i.e. Python-only, no Flutter method for this proto, that was my mistake originally).
- give it a proto override file at `static/include/app/apis/overrides/protos/mltraining.SubmitCustomTrainingJob.md`. Oh! I already did I think.
- run autogen with `python3 .github/workflows/update_sdk_methods.py -v mltraining` and check to be sure `static/include/app/apis/generated/mltraining-table.md` and `static/include/app/apis/generated/mltraining.md` were updated as expected!

Reply:
Just ran it to test to be sure: looks good, should be an easy one!

Reply:
Awesome thank you for all the help!